src/forms/_checkbox.scss

```diff
@@ -0,0 +1,33 @@
+.checkbox {
+  display: none;
+
+  + .checkbox__label::before {
+    content: '';
+    position: absolute;
+    top: 4px;
+    left: 0;
+    width: 16px;
+    height: 16px;
+    border: 1px solid color(gray--light3);
+    cursor: pointer;
+    text-align: center;
+    line-height: 0.5;
+  }
+
+  &.error {
+    & + .checkbox__label::before {
+      border-color: color(red);
+    }
+  }
+
+  &:checked {
+    border-color: color(blue);
+
+    & + .checkbox__label::before {
+      content: '\d7';
+      color: color(blue);
```

Comment:
Our blue should be specified as `primary` to avoid confusion with the default CSS color named values.